models/fivetran_platform.yml

```diff
@@ -53,17 +53,11 @@ models:
         description: > 
           The number of schema changes (creating tables or schemas and altering tables) in the past 30 days.
 
-      - name: errors_since_last_completed_sync
-        description: >
-          Aggregated line-separated list of error messages (in JSON format) raised synce the last 
-          sync completion. Included by default for non-SQL Server targets, but can be disabled by setting
-          the `fivetran_platform_using_sync_alert_messages` var to False.
+      - name: number_errors_since_last_completed_sync
+        description: Numeric metric indicating the number of error messages raised sync the last sync completion.
```

Comment:
```suggestion
      - name: number_errors_since_last_completed_sync
        description: Numeric metric indicating the number of error messages raised since the last sync completion.
```